assets/css/v2/on_bus/no_data.scss

```diff
@@ -0,0 +1,27 @@
+@use "CSS/colors";
+
+.no-data {
+  position: relative;
+  width: 552px;
+  height: 354px;
+  margin-bottom: 23px;
+  margin-left: 36px;
+}
+
+.no-data__header {
+  position: relative;
+  top: 23px;
+  width: 90px;
+  height: 90px;
+  background-color: colors.$warm-neutral-90;
+}
+
+.no-data__content {
+  position: relative;
+  top: 28px;
+  font-size: 59px;
+  font-weight: 700;
+  line-height: 66px;
+  color: colors.$cool-black-15;
+  letter-spacing: 0;
```

Comment:
This should be equivalent to the default and so not necessary to specify (it does have behavior differences from the actual default value of `normal`, but those should only come into play when text is being justified).